src/main/scala/com/exasol/cloudetl/storage/StorageProperties.scala

```diff
@@ -44,11 +44,8 @@ class StorageProperties(
    * requirements.
    */
   final def getDeltaFormatLogStoreClassName(): String = getStoragePathScheme() match {
-    case "abfs" | "abfss" =>
-      throw new UnsupportedOperationException(
-        "Delta format LogStore API is not supported in Azure Data Lake Gen2 storage yet."
-      )
-    case "adl" => "org.apache.spark.sql.delta.storage.AzureLogStore"
+    case "abfs" | "abfss" => "org.apache.spark.sql.delta.storage.AzureLogStore"
+    case "adl"            => "org.apache.spark.sql.delta.storage.AzureLogStore"
```

Comment:
Why are they separated?

Reply:
This better, since they refer two different storage systems. 
(the additional 's' stand for the secure version, but adl does not have one).